project-base/tests/FrontendApiBundle/Test/GraphQlWithLoginTestCase.php

```diff
@@ -85,9 +91,10 @@ private function getResponseForQuery(string $query, array $variables, array $cus
     /**
      * @param string|null $customerUserEmail
      * @param string|null $customerUserPassword
+     * @param array $customServer
      * @return string
      */
-    private function getAccessToken(?string $customerUserEmail = null, ?string $customerUserPassword = null): string
+    private function getAccessToken(?string $customerUserEmail = null, ?string $customerUserPassword = null, array $customServer = []): string
```

Comment:
`$customServer` is not used in this context

Reply:
Oh, thank you, fixed!